spec/lib/remote_translations/microsoft/available_locales_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+describe RemoteTranslations::Microsoft::AvailableLocales do
+  describe ".available_locales" do
+    context "app locales that need to be mapping" do
```

Comment:
I guess we could remove this line, since there's only one test here.

If we decide to keep this line, there seems to be a typo:

```suggestion
    context "app locales that need to be mapped" do
```